server/backend/users/tests/test_serializers.py

```
@@ -0,0 +1,34 @@
+import pytest
+from rest_framework.exceptions import ValidationError
+from django.contrib.auth import get_user_model
+from users.models import Instructor
+from users.serializers import InstructorSerializer
+
+User = get_user_model()
+
+@pytest.fixture
+def create_instructor():
+    user = User.objects.create_user(username="john_doe", password="password123", role="instructor")
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)